### ISSUE TITLE
website renaming?

### ISSUE THREAD

**Issue by ebeshero** [lmcneil7/teen-titans#3]
@lmcneil7 @smdunn921 I was busy setting up your website at I think the very moment you were deliberating over what to call it yesterday, so I didn't see your issue post at the time. It did turn up in my e-mail though, and I saw it eventually. If you want to rename the site, we can do that--let me know. Probably best we do this fast before you start building the site.

The current name is http://teentitans.newtfire.org 

For my part, I'd say keeping this is good because it helps make your site easily findable on the web for those interested in the series / comics!  

Here's the list of alternative names you'd posted:

website names:
-tt
-teent
-titanst
-titansteeth
-tnt
-titansgo
-teentitansgo (i know this is longer but eh)
-newteentitans (it's the name of the comic the show is based off)
-teetit
-teentitanz

If you have a strong preference to change the name, let me know! Otherwise you can just close this issue. :-)

**Comment by lmcneil7** [comment]
hey dr.b, yeah we had no intention on changing the name so teentitans is good! Professor Bondar had told us we should open an issue just to let you know what we wanted to name the website; at the time I thought he meant that teentitans was too long, so I came up with alternates to make it shorter. However, he posted an issue about how he thought our name was gonna be teentitans.newtfire.org and so then we didn't need the list anymore. I posted the issue just in case @smdunn921 wanted to look at the alternates, but then realized that the issue was kind of unnecessary. That's why I deleted it, but teentitans is good! Thanks for asking though.
